integration/docker/entrypoint.sh

```diff
@@ -159,7 +159,16 @@ function setup_for_dynamic_non_root {
       mkdir -p /journal
       chown -R ${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} /opt/* /journal
       chmod -R g=u /opt/* /journal
-      exec su ${ALLUXIO_USERNAME} -c "/entrypoint.sh ${ARGS}"
+      # Chmod the dirs of tiered stores for alluxio worker
+      # to ensure write permission for non-root user.
+      if [[ "$1" == "worker" || "$1" == "worker-only" ]]; then
+        echo "$ALLUXIO_JAVA_OPTS" | tr ' ' '\n' | \
+          grep "alluxio.worker.tieredstore.level[0-9].dirs.path" | \
+          cut -d '=' -f 2 | \
+          grep -Ev "^$" | \
+          xargs -I {} chmod -R 777 {}
```

Comment:
I tested the script and it still doesn't seem to work

I created directores `a1` and `a2` in my working directory, the same place I ran this command.
```
#!/usr/bin/env bash
set -x
tvar="-Dalluxio.worker.tieredstore.level0.dirs.path=a1,a2"

echo "$tvar" | tr ' ' '\n' | \
grep "alluxio.worker.tieredstore.level[0-9].dirs.path" | \
cut -d '=' -f 2 | \
grep -Ev "^$" | \
xargs -I {} chmod -R 777 {}
```

output:

```
+ tvar=-Dalluxio.worker.tieredstore.level0.dirs.path=a1,a2
+ echo -Dalluxio.worker.tieredstore.level0.dirs.path=a1,a2
+ tr ' ' '\n'
+ grep 'alluxio.worker.tieredstore.level[0-9].dirs.path'
+ cut -d = -f 2
+ grep -Ev '^$'
+ xargs -I '{}' chmod -R 777 '{}'
chmod: cannot access 'a1,a2': No such file or directory
```

Reply:
Are you testing on the latest version? 
I tried and it is working fine.
script:
```bash
#!/usr/bin/env bash
set -x
tvar="-Dalluxio.worker.tieredstore.level0.dirs.path=a1,a2"
echo "${tvar}" | \
  tr ' ' '\n' | \
  grep "alluxio.worker.tieredstore.level[0-9].dirs.path" | \
  cut -d '=' -f 2 | \
  tr ',' '\n' | \
  grep -Ev "^$" | \
  xargs -I {} chmod -R 777 {}
```
output:
```bash
+ tvar=-Dalluxio.worker.tieredstore.level0.dirs.path=a1,a2
+ echo -Dalluxio.worker.tieredstore.level0.dirs.path=a1,a2
+ tr ' ' '\n'
+ grep 'alluxio.worker.tieredstore.level[0-9].dirs.path'
+ cut -d = -f 2
+ tr , '\n'
+ grep -Ev '^$'
+ xargs -I '{}' chmod -R 777 '{}'
chmod: a1: No such file or directory
chmod: a2: No such file or directory
```